--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -3,11 +3,6 @@
 execute_process(COMMAND python2 -c "from distutils.sysconfig import get_python_lib; print get_python_lib()"
     OUTPUT_VARIABLE PYTHON_PACKAGE_DIR OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-message("===== ${AUTOPILOT_TESTS_DIR}")
 message("===== ${PYTHON_PACKAGE_DIR}")
 
-install(DIRECTORY ${AUTOPILOT_TESTS_DIR}
-    DESTINATION ${PYTHON_PACKAGE_DIR}
-    )
-
 add_subdirectory(unittest)
