--- a/chewieui/Ubuntu/ChewieUI/CMakeLists.txt
+++ b/chewieui/Ubuntu/ChewieUI/CMakeLists.txt
@@ -37,7 +37,7 @@
     qmldir
 )
 
-set(QT_IMPORTS_DIR ${_qt5Qml_install_prefix}/${CMAKE_INSTALL_LIBDIR}/qt5/qml)
+set(QT_IMPORTS_DIR ${_qt5Qml_install_prefix}/${CMAKE_INSTALL_LIBDIR}/qt/qml)
 set(CHEWIEUI_INSTALL_PREFIX "${QT_IMPORTS_DIR}/Ubuntu/ChewieUI")
 
 install(FILES ${CHEWIEUI_SRC} DESTINATION ${CHEWIEUI_INSTALL_PREFIX})
